source/GooglePlayServicesProject.cshtml

```diff
@@ -146,14 +146,20 @@
   <ItemGroup>
     @foreach (var art in @Model.MavenArtifacts)
     {
+      if (@Model.NuGetPackageId == "Xamarin.Firebase.ProtoliteWellKnownTypes" )
```

Comment:
I don't think this is related to this PR?

Reply:
the whole `JavaSourceJar` and `JavaDocJar` is currently commented out. I left it because tooling has issues with few packages and commented part is actually list of problematic packages (preparing issue for that).